docs/platforms/apple/common/configuration/http-client-errors.mdx

```diff
@@ -73,6 +73,30 @@ SentrySDK.start { options in
 }];
 ```
 
+### GraphQL Operation Tracking
+
+When `enableGraphQLOperationTracking` is enabled, the SDK extracts the GraphQL operation name from HTTP requests that have `Content-Type: application/json` and contain a JSON body with an `operationName` field. The operation name is then included in the error event's context as `graphql.operation_name`, making it easier to identify which GraphQL operation failed.
```

Comment:
`m`: I think we should link here to docs/platforms/apple/common/configuration/graphql-operation-tracking.mdx